config/astral.yml

```diff
@@ -1,9 +1,9 @@
 shared:
   vault_addr: <%= ENV["VAULT_ADDR"] %>
   vault_token: <%= ENV["VAULT_TOKEN"] %>
-  vault_cert_path: "pki_int/issue/learn"
+  vault_cert_path: <%= ENV["SHARED_VAULT_CERT_PATH"] %>
   jwt_signing_key: <%= ENV["JWT_SIGNING_KEY"] %>
-  cert_ttl: <%= 24.hours.in_seconds %>
+  cert_ttl: <%= ENV["SHARED_CERT_TTL"] %>
```

Comment:
i think this ENV substitution is causing test failures since it is now string and validation wants an int -- i'd suggest: `<%= ENV["SHARED_CERT_TTL"].to_i %>`